README.md

```diff
@@ -233,32 +250,35 @@ $ tree my-component
 First, install the cli and then use `manifest-gen` to do this.
 
 ``` shell
-$ cd modality-probe-cli
+$ cd ../modality-probe-cli
 $ cargo install --path .
-$ cd ../
+$ cd ../modality-probe-capi
 $ modality-probe manifest-gen \
-    --lang rust \
-    --file-extension rs \
+    --lang c \
+    --file-extension c \
     --component-name example-component \
     --output-path example-component \
-    examples/rust-example
+    examples/c-example
 ```
 
 Next, we'll want to generate the source code that gives those symbols
 we discussed in the code snippet examples in the previous section
 their definitions. To do that, we'll use `header-gen`:
 
 ```shell
+$ mkdir -p ../examples/c-example/include
 $ modality-probe header-gen \
-    --lang rust \
-    --output-path examples/rust-example/src/component_definitions.rs \
+    --lang c \
+    --output-path ../examples/c-example/include/component_definitions.h \
     example-component
 ```
 
-NOTE: It can be helpful to have the manifest & header generation tools run as
+NOTE: It can be helpful to have the manifest and header generation tools run as
 part of your regular build process to automatically pick up changes to
 your instrumentation or alert you to potential issues in your instrumentation.
-To do this you can include this process in your crate's `build.rs` file.
+
+If you are targeting a Rust application, you can do this by including this
```

Comment:
If the whole rest of the thing is gonna be in C this is kind of weirdly dropped in. Let's just remove it and save it for when we do a Rust API doc, or as a stretch get an example from Jon about how to do this for C.

Reply:
:+1: Sure, I was leaving this in on the principle of "don't remove any decent/cool functionality" as followed in keeping the defunct `visualize` command in.

:question: @paelionmoss does this fall below the threshold of "decent/cool functionality"? I.e., is this a simple script that doesn't rely on anything Modality does for you?

Reply:
Yeah this isn't Modality functionality, it's instructions for calling Modality's commands (which we've already told you about) as part of your build.

Reply:
:+1: roger, will remove.